nixos/modules/services/web-apps/peertube.nix

```diff
@@ -282,6 +294,18 @@ in {
             prevent this.
           '';
       }
+      { assertion = cfg.secrets.secretsFile != null;
+          message = ''
+            <option>services.peertube.secrets.secretsFile</option> needs to be set.
+          '';
+      }
+      { assertion = cfg.secrets.secretsFile == null || !lib.hasPrefix builtins.storeDir cfg.secrets.secretsFile;
+          message = ''
+            <option>services.peertube.secrets.secretsFile</option> points to
+            a file in the Nix store. You should use a quoted absolute path to
+            prevent this.
+          '';
+      }
```

Comment:
We do not need or want to assert that. It is common knowledge that this is insecure.